src/lxc/attach.c

```diff
@@ -658,14 +678,14 @@ static bool fetch_seccomp(struct lxc_container *c, lxc_attach_options_t *options
 
 	/* Remove current setting. */
 	if (!c->set_config_item(c, "lxc.seccomp.profile", "") &&
-	    !c->set_config_item(c, "lxc.seccomp", "")) {
```

Comment:
@brauner, I've been meaning to ask whether the single-line-if exception in the coding style also holds for cases like this where the condition spans across multiple lines?

Reply:
Yeah, it does. )

Reply:
@brauner in case of this, brace is needed?

Reply:
No, no braces needed.